examples/reproducer/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+tesseract --loglevel debug build .
+
+docker system prune --force
+
+tesseract run reproducer apply '{"inputs":{}}' --output-path outputs
```

Comment:
if we're going to commit this example let's name it something more specific, like `process-pool-executor`